index/miner/miner.go

```diff
@@ -153,10 +154,11 @@ func (mi *MinerIndex) Close() error {
 // miners.
 func (mi *MinerIndex) start() {
 	defer func() { mi.finished <- struct{}{} }()
-	n, err := mi.api.ChainNotify(mi.ctx)
```

Comment:
Stop using `ChainNotify` for _listening_ to the chain.
I've been thinking about this for quite some time since puts a lot of stress. It notifies about every possible forks that might happen in 10-20s time frame... we're only interested in the heaviest tipset known and let time (and lotus) decide which are the heaviest ones.

Switched to waiting the avg tipset time and just asking for the heaviest current tipset (which is what we need). Mini forks or possible future forks aren't interesting.  We don't need to be at the bleeding-edge of that.